assets/app.js

```diff
@@ -338,6 +338,39 @@ new Vue({
         }
     },
     computed: {
+        validateCacheMaterial: function () {
+            active_languages = this.activeLanguagesId
+            this.products.forEach(function (product) {
```

Comment:
can we do it when we add/get product(s)?

Reply:
`foreach` cannot use `break`

Reply:
it active when we have data in products and open materials page.